core/controllers/cron_test.py

```diff
@@ -447,6 +449,223 @@ def test_email_not_sent_if_reviewer_ids_is_empty(self) -> None:
+                self.CURRICULUM_ADMIN_EMAIL, is_super_admin=True)
+            swap_platform_parameter_value = self.swap_to_always_return(
+                platform_feature_services,
+                'get_platform_parameter_value',
+                True
+            )
+
+            with self.cannot_send_emails, self.testapp_swap, swap_platform_parameter_value: # pylint: disable=line-too-long
+                self.get_json(
+                    '/cron/mail/reviewers/new_contr' +
+                    'ibutor_dashboard_suggestions')
+
+            mock_send.assert_not_called()
+            self.logout()
+
+    def test_email_sent_to_reviewer_if_sending_reviewer_emails_is_enabled(self) -> None: # pylint: disable=line-too-long
+        # Here we use object because we need to spy on
+        # send_reviewer_notifications method and assert
+        # if it's being called.
+        with mock.patch.object(
+            email_manager, 'send_reviewer_notifications',
+            new_callable=mock.Mock
+        ) as mock_send:
+            self.login(
+                self.CURRICULUM_ADMIN_EMAIL, is_super_admin=True)
+            swap_platform_parameter_value = self.swap_to_always_return(
+                platform_feature_services,
+                'get_platform_parameter_value',
+                True
+            )
+
+            with self.can_send_emails, self.testapp_swap, swap_platform_parameter_value: # pylint: disable=line-too-long
+                self.get_json(
+                    '/cron/mail/reviewers/new_cont' +
+                    'ributor_dashboard_suggestions')
+            mock_send.assert_called_once_with(
+                {'en': [self.reviewer_id]},
+                {'en': [mock.ANY]}
+            )
+
+            self.logout()
+
+    def test_email_not_sent_if_reviewer_ids_is_empty(self) -> None:
+        # Here we use object because we need to spy on
+        # send_reviewer_notifications method and assert
+        # if it's being called.
+        with mock.patch.object(
+            email_manager, 'send_reviewer_notifications',
+            new_callable=mock.Mock
+        ) as mock_send:
+            self.login(
+                self.CURRICULUM_ADMIN_EMAIL, is_super_admin=True)
+            swap_platform_parameter_value = self.swap_to_always_return(
+                platform_feature_services,
+                'get_platform_parameter_value',
+                True
+            )
+
+            user_services.remove_translation_review_rights_in_language(
+                self.reviewer_id, self.language_code)
+            user_services.remove_translation_review_rights_in_language(
+                self.reviewer_id, 'hi'
+            )
+            with self.can_send_emails, self.testapp_swap, swap_platform_parameter_value: # pylint: disable=line-too-long
+                self.get_json(
+                    '/cron/mail/reviewers/new_contr' +
+                    'ibutor_dashboard_suggestions')
+            mock_send.assert_called_once()
```

Comment:
Don't need this. `asserted_called_once_with` checks both the args and that the function was called only once.

Reply:
Done, Thanks.